esp_driver_sccb/sccb_i2c/include/sccb_i2c.h

```diff
@@ -0,0 +1,36 @@
+/*
+ * SPDX-FileCopyrightText: 2024 Espressif Systems (Shanghai) CO LTD
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "driver/i2c_master.h"
+#include "sccb_types.h"
+
+/**
+ * @brief ESP SCCB I2C configuration
+ */
+typedef struct {
+    i2c_port_num_t i2c_port;              ///< I2C port number, `-1` for auto selecting
+    gpio_num_t sda_io_num;                ///< GPIO number of I2C SDA signal, pulled-up internally
+    gpio_num_t scl_io_num;                ///< GPIO number of I2C SCL signal, pulled-up internally
+    i2c_addr_bit_len_t dev_addr_length;   ///< Select the address length of the slave device
+    uint16_t device_address;              ///< I2C device raw address. (The 7/10 bit address without read/write bit)
+    uint32_t scl_speed_hz;                ///< I2C SCL line frequency
+    i2c_clock_source_t clk_source;        ///< Clock source of I2C master bus, channels in the same group must use the same clock source
+} esp_sccb_i2c_config_t;
```

Comment:
does this driver follow bus-device model? If so, what does the sccb represents? If it's a bus driver, I didn't expect there is a device address in the same configuration structure. BTW, device address may reside in a camera sensor's own driver package in case different camera needs special initialization commands.